sdk/identity/azure-identity/APP_REGISTRATION.md

```diff
@@ -0,0 +1,138 @@
+# Application Registration and Role Assignment
+
+## Prerequisites
+
+- An Azure account that has an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
```

Comment:
```suggestion
- An Azure account that has an active subscription. [Create an account for free](https://azure.microsoft.com/free/python).
```

Reply:
@scottaddie 

I consider creating an aka link and share it across all languages.

How about making it language agnostic by just using https://azure.microsoft.com/free/?